doc/source/whatsnew/v0.21.0.txt

```diff
@@ -197,6 +197,7 @@ Other API Changes
   raise an ``AttributeError`` rather than a ``ClosedFileError`` (:issue:`16301`)
 - :func:`read_csv` now treats ``'null'`` strings as missing values by default (:issue:`16471`)
 - :func:`read_csv` now treats ``'n/a'`` strings as missing values by default (:issue:`16078`)
+- :func:`read_json` now accepts a ``chunksize`` parameter that can reduce memory usage when ``lines=True``. (:issue:`17048`)
```

Comment:
Can you put this in the "Other Enhancements" section?

Reply:
update for this